server/liveblog/blogs/utils.py

```diff
@@ -10,8 +10,10 @@ def check_media_storage():
         raise MediaStorageUnsupportedForBlogPublishing()
 
 
-def get_blog_path(blog_id):
-    return 'blogs/%s/index.html' % (blog_id)
+def get_blog_path(blog_id, theme=None, output_id=None):
+    return 'blogs/{}/{}{}index.html'.format(blog_id,
+                                            '{}/'.format(theme) if theme else '',
```

Comment:
yes, this is how was thought at the beginning when multiple themes where in place.
now we can have another behavior but this isn't breaking that.